File: 프로그래머스/lv0/120839. 가위 바위 보/가위 바위 보.py
def solution(rsp):
    answer = {'2':'0', '0':'5', '5':'2'}
    return ''.join(answer[i] for i in rsp)

# 딕셔너리 key==value 형태로 만들기
# rsp가 key니까 해당하는 value값 출력
# answer.get(key) 이렇게 value 값 출력할 수도 있음

File: 프로그래머스/lv0/120839. 가위 바위 보/가위 바위 보.java
class Solution {
    public String solution(String rsp) {
        String answer = "";
        String[] str = rsp.split("");
        for (int i=0;i<str.length;i++){
            if(str[i].equals("2")){
                answer += "0";
            }else if(str[i].equals("0")){
                answer += "5";
            }else{
                answer += "2";
            }
        }
    return answer;
    }
}